model/cloud.go

```diff
@@ -45,9 +45,9 @@ type CloudCustomer struct {
 // CloudCustomerInfo represents editable info of a customer.
 type CloudCustomerInfo struct {
 	Name             string `json:"name"`
-	Email            string `json:"email"`
-	ContactFirstName string `json:"contact_first_name"`
-	ContactLastName  string `json:"contact_last_name"`
+	Email            string `json:"-"`
```

Comment:
Would `omitempty` work here?

```suggestion
	Email            string `json:"email,omitempty"`
```